Fix .Site.LastChange doc (#2237)
gohugoio/hugoDocs: content/en/variables/site.md

: an ordered list (ordered by defined weight) of languages.

.Site.LastChange

> : a [time.Time](https://godoc.org/time#Time) representing the date/time of the most recent change to your site. This string is based on the [`date` variable in the front matter](/content-management/front-matter) of your content pages.

Reviewer: ```suggestion
: a [time.Time](https://godoc.org/time#Time) value representing the date/time of the most recent change to your site. 
```
